core/utils.py

```diff
@@ -326,10 +327,7 @@ def convert_png_binary_to_data_url(content: Union[str, bytes]) -> str:
     # So we have casted content to be bytes.
     content = python_utils.convert_to_bytes(content)
     if imghdr.what(None, h=content) == 'png':
-        return '%s%s' % (
-            PNG_DATA_URL_PREFIX,
-            python_utils.url_quote(base64.b64encode(content))  # type: ignore[no-untyped-call]
-        )
+        return '%s%s' % (PNG_DATA_URL_PREFIX, urllib.parse.quote(base64.b64encode(content)))
```

Comment:
```suggestion
        return '%s%s' % (
            PNG_DATA_URL_PREFIX, urllib.parse.quote(base64.b64encode(content))
        )
```

Reply:
Btw, you can notice this in the failing [lint check](https://github.com/oppia/oppia/runs/3778131466#step:8:569)

Reply:
@vojtechjelinek Done